src/core/messages.ts

```diff
@@ -134,6 +134,32 @@ export interface SendMessageParams {
   headers?: MessageHeaders;
 }
 
+/**
+ * Params for updating a message. It accepts all parameters that sending a
+ * message accepts.
+ *
+ * Note that updating a message replaces the whole previous message, so all
+ * metadata and headers that should be kept must be set in the update request,
+ * or they will be lost.
+ */
+// eslint-disable-next-line @typescript-eslint/no-empty-object-type
+export interface UpdateMessageParams extends SendMessageParams {}
+
+/**
+ * Allows setting optional details about the update operation itself.
+ */
+export interface UpdateMessageDetails {
+  /**
+   * An optional description for the update operation.
```

Comment:
If we decided not to merge the two interfaces, could we make the typedocs message consistent? Happy to keep this text or the other above

Reply:
Merged the interfaces